Client/overrides/scripts/ConflictFix.zs

```diff
@@ -4,6 +4,11 @@ import crafttweaker.item.IIngredient;
 
 print("--- loading ConflictFix.zs ---");
 
+#Remove multishot
+mods.enderio.Enchanter.removeRecipe(<enchantment:cofhcore:multishot>);
+mods.thermalexpansion.Enchanter.removeRecipe(<minecraft:book>, <minecraft:arrow>);
```

Comment:
@criscky Won't this remove all recipes, or is this recipe just for the multishot recipe?

Reply:
only the recipe, adam tried to remove the enchant, but because of how cofh is coded it doesnt "un-register" the enchantment, and because of mod interactions the enchatment can still be "made"